jobs/integration/validation.py

```diff
@@ -814,6 +819,9 @@ async def cuda_test():
         timeout_insec=20 * 60,
     )
 
+    # Cleanup the deployed nvidia-gpu-operator charm
+    k8s_model.remove_application("nvidia-gpu-operator")
```

Comment:
this is cool... but it should be noted that if the test fails, the nvidia-gpu-operator will still be deployed.  That may not be too bad though b/c we'd get a juju-crashdump of the thing and we could see what's wrong with the model